libbeat/template/config.go

```diff
@@ -1,12 +1,17 @@
 package template
 
 type TemplateConfig struct {
-	Enabled      bool                   `config:"enabled"`
-	Name         string                 `config:"name"`
-	Fields       string                 `config:"fields"`
-	Overwrite    bool                   `config:"overwrite"`
-	OutputToFile string                 `config:"output_to_file"`
-	Settings     map[string]interface{} `config:"settings"`
+	Enabled      bool             `config:"enabled"`
+	Name         string           `config:"name"`
+	Fields       string           `config:"fields"`
+	Overwrite    bool             `config:"overwrite"`
+	OutputToFile string           `config:"output_to_file"`
+	Settings     TemplateSettings `config:"settings"`
+}
+
+type TemplateSettings struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type TemplateSettings should have comment or be unexported